### PR TITLE
7135 add filters

### PR DESCRIPTION
**Pull request by kris45**

### CODE REVIEW

File: conference/react-js/src/reducers/talks.js [hunk 1]
```diff
@@ -9,17 +9,19 @@ const talks = (state = [], action) => {
     return payload;
   } else if (type === APPLY_FILTERS) {
     const {
-      filter: { topic, status },
+      filter: { conferenceName, title, status },
```
Comment (ScreamCat1) [inline conference/react-js/src/reducers/talks.js]: We need some else functionality for filters. This reducer should take different numbers of filter fields

File: conference/react-js/src/containers/Talks/index.jsx [hunk 2]
```diff
@@ -124,7 +121,7 @@ class Talks extends Component {
 
   handleResetFiltersClick = () => {
     this.setState({
-      filter: { topic: '', status: '' },
+      filter: { conferenceName: '', title: '', status: '' },
```
Comment (ScreamCat1) [inline conference/react-js/src/containers/Talks/index.jsx]: We want use this function to another data (in My talks and Talks components). So number of fields will be different

File: conference/react-js/src/containers/Talks/index.jsx [hunk 3]
```diff
@@ -9,19 +9,16 @@ import TalksTable from '../../components/Talks/TalksTable/TalksTable';
...
+import { talk, talksStatus } from '../../constants/backend-url';
 
 class Talks extends Component {
   constructor(props) {
     super(props);
     this.state = {
       talks: [],
-      filter: { topic: '', status: '' },
-      listOfTopics: [],
+      filter: { conferenceName: '', title: '', status: '' },
```
Comment (ScreamCat1) [inline conference/react-js/src/containers/Talks/index.jsx]: Same problem

File: conference/react-js/src/containers/Talks/FilterForm/index.jsx [hunk 4]
```diff
@@ -1,52 +1,48 @@
...
-
-  chooseTopic = (item) => {
-    this.setState({ topicValue: item });
-  }
-
-  resetTopics = () => {
-    this.props.handleResetFiltersClick();
-    this.setState({ topicValue: '' });
-  }
+  setStatus = status => (status.map(element =>
```
Comment (SupraVlad) [inline conference/react-js/src/containers/Talks/FilterForm/index.jsx]: as I've noticed, this method returns a list of elements(options) so the name `getOptions` will mach better

File: conference/react-js/src/containers/Talks/FilterForm/index.jsx [hunk 5]
```diff
@@ -1,52 +1,48 @@
...
-      onChangeFilter,
-      handleFilterClick } = this.props;
+    const { status, onChangeFilter, handleFilterClick,
+      handleResetFiltersClick } = this.props;
     return (
       <div className="my-talk-settings">
         <form className="my-talk-settings__filters">
           <p className="my-talk-settings__title">filter by:</p>
           <div className="my-talk-settings__select-wrapper">
             <label
```
Comment (SupraVlad) [inline conference/react-js/src/containers/Talks/FilterForm/index.jsx]: I remember we have custom component for label+input. reuse it, please

**Review by SupraVlad — CHANGES_REQUESTED** [review]

**Review by SupraVlad — APPROVED** [review]

**Review by asanametove — APPROVED** [review]

**Pull request merged**